frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -259,7 +259,7 @@ describe("scenarios > question > notebook", () => {
       popover().within(() => {
         cy.get("[contenteditable='true']").type(
           // reference joined question by previously set alias
-          "[13649 → Sum of Rating] / [Sum of Rating]",
```

Comment:
@nemanjaglumac Not sure if this change makes sense or not but this is what showed up and if you click `sum` it works as expected. I'm guessing it has something to do with the fact that the Cypress tests don't save metadata when they save the questions the way the FE client normally does

Reply:
Could be.
Maybe this has something to do with the work that @ariya is doing around custom expressions? If I'm not wrong, there's some name "normalization" going on.